pwiz_tools/Skyline/SkylineGraphs.cs

```diff
@@ -2998,6 +3037,14 @@ private void closeAllChromatogramsMenuItem_Click(object sender, EventArgs e)
             }
         }
 
+        /// <summary>

```

Comment:
Normally, we do this by extracting a public function and having the event handler call that, rather than adding a public function that calls the event handler. This is safer and avoids the confusion of someone trying to use the passed in objects when they are null (for your test method).